src/bundle/Resources/public/scss/fieldType/edit/_eznoneditable.scss

```diff
@@ -1,19 +1,14 @@
-.ez-field-edit--eznoneditable {
-    display: flex;
+.ibexa-field-edit--eznoneditable {
```

Comment:
should this even be in separate file? It's just modifier to other class, that has 14 lines...

Reply:
All modifiers of field types are in separate files so yes.